.gitignore

```diff
@@ -49,6 +49,7 @@ scripts/*.csv
 
 # -- Tools
 .coverage
+.idea
```

Comment:
It's often considered as a good practice to avoid this on a per-project basis, but rather on a global user configuration (_i.e._ `$HOME/.gitignore`)